filecoin/functions/piece-cid-compute.js

```diff
@@ -0,0 +1,97 @@
+import { S3Client } from '@aws-sdk/client-s3'
+import * as Sentry from '@sentry/serverless'
+
+import { computePieceCid } from '../index.js'
+import { createPieceTable } from '../tables/piece.js'
+
+Sentry.AWSLambda.init({
+  environment: process.env.SST_STAGE,
+  dsn: process.env.SENTRY_DSN,
+  tracesSampleRate: 1.0,
+})
+
+const AWS_REGION = process.env.AWS_REGION || 'us-west-2'
+
+/**
+ * Get EventRecord from the SQS Event triggering the handler
+ *
+ * @param {import('aws-lambda').SQSEvent} event
+ */
+async function computeHandler (event) {
+  const {
+    pieceTableName,
+  } = getEnv()
+
+  const record = parseEvent(event)
+  if (!record) {
+    throw new Error('Unexpected sqs record format')
+  }
+
+  const s3Client = new S3Client({ region: record.bucketRegion })
+  const pieceTable = createPieceTable(AWS_REGION, pieceTableName)
+
+  const { error, ok } = await computePieceCid({
+    record,
+    s3Client,
+    pieceTable,
+  })
+
+  if (error) {
```

Comment:
Log it so we can get stack if we ever need to!

```suggestion
  if (error) {
    console.error(error)
```